web.browser/src/components/PostToolbar/PostToolbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { Toolbar, ToolbarTitle } from 'material-ui/Toolbar';
+import FlatButton from 'material-ui/FlatButton';
+import styles from './styles.css';
+import { sortNewest, sortPopular } from '../../redux/actions';
+
```

Comment:
Grouping related imports will help you stay organised. Decide how you're going to group and make sure you're consistent across all files
eg. In this file: 

```
import React from 'react';

import { connect } from 'react-redux';
import { sortNewest, sortPopular } from '../../redux/actions';
 
import { Toolbar, ToolbarTitle } from 'material-ui/Toolbar';
import FlatButton from 'material-ui/FlatButton';

import styles from './styles.css';
```